companionship.md

```diff
@@ -0,0 +1,80 @@
+
+﴾ Mohammed SAWS is the Messenger of Allah, and His companions are strict with the non-believers and merciful between eachother. You see them in rukuu' and sujood, seeking Allah's bounty and pleasure...
+
+Their sign is in their faces from the effects of sujood. That is their description in the Torah. Their description in the Injeel is like a plant that has given its offshoots, and it strengthens them so they grow firm and stand on their stalks, delighting the sowers. So that Allah enrages the disbelivers by the belivers.
+
+Allah has promised the ones who have believed and done righteous deeds forgiveness and great reward. ﴿
+
+## Allah SWT also says in Surah Ali Imran, verse 28
+
+<details>
+  <summary> expand </summary>
+
+  We will not translate this
+
+  ﴿لا يَتَّخِذِ المُؤمِنونَ الكافِرينَ أَولِياءَ مِن دونِ المُؤمِنينَ وَمَن يَفعَل ذلِكَ فَلَيسَ مِنَ اللَّهِ في شَيءٍ إِلّا أَن تَتَّقوا مِنهُم تُقاةً وَيُحَذِّرُكُمُ اللَّهُ نَفسَهُ وَإِلَى اللَّهِ المَصيرُ﴾
+</details>
+<br>
+
+## Hadith
+
+حَدَّثَنَا  مُحَمَّدُ بْنُ بَشَّارٍ ، قَالَ : حَدَّثَنَا  أَبُو عَامِرٍ ،  وَأَبُو دَاوُدَ ، قَالَا : حَدَّثَنَا  زُهَيْرُ بْنُ مُحَمَّدٍ ، قَالَ : حَدَّثَنِي  مُوسَى بْنُ وَرْدَانَ ، عَنْ  أَبِي هُرَيْرَةَ  قَالَ : قَالَ رَسُولُ اللَّهِ صَلَّى اللَّهُ عَلَيْهِ وَسَلَّمَ : " الرَّجُلُ عَلَى دِينِ خَلِيلِهِ، فَلْيَنْظُرْ أَحَدُكُمْ مَنْ يُخَالِلُ ".
+ هَذَا حَدِيثٌ حَسَنٌ غَرِيبٌ.
+
+On the authority of Abu Huraira RA, who said, The Messange of Allah SAWS said "A man is on the religion of his (close) friend, so look at who you take as close friends" (Sunan of Abi Dawood and Sunan Al Tirmidi)
+
+## Break & Sit
+
+﴿فَقُلتُ استَغفِروا رَبَّكُم إِنَّهُ كانَ غَفّارًا﴾
+
+<hr>
+
+# The second half
+
+بسم الله، والحمدلله، والصلاة والسلام على رسول الله، وعلى اله وصحبه ومن والاه
+
+As we see there is lots in the Quran and Sunnah that highlights the importance of righteous and Muslim compnaionship. We see that in Surah Ali Imran, Allah SWT ties salvation from Hell Fire to Islamic Brotherhood.
+
+And we see that in Surah Al Fath, Allah SWT says that the people with The Rasool SAWS are merciful and gentle amongst themselves.
+
+When we look at the state of the Muslim Ummah, we see that, generally, we are not adhering to what Allah SWT has prescribed. We see brothers cut friendships because their countries are warring, we see _Muslim countries_ warring over nothing. The izzah that the Muslims had in the early days of the Ummah were in part due to Muslims duty to one another, and our brotherhood. We were once one nation, one body, and when we cared for eachother, Allah SWT gave us victory, peace, and stability. Now we care about borders that the west has imposed on us, and all we see is strife and pain. We need to keep the ties between us, start on a grass roots level of having rahma between us, and ان شاء الله we will be given victory, peace, and stability once more.
```

Comment:
Friendships breaking due to warring countries???  Not on a college campus.  Keep the talk relevant to students. 

Khutbah is very very short, unless you plan to really talk about the ayaat and dive into the topic more than you have put down, this will be over in no time.